scripts/flake8_py_version_check.py

```diff
@@ -0,0 +1,36 @@
+import json
+import re
+import subprocess
+
+
+def main():
+    # find a line in pyproject.toml that looks like `python-requires = "..."`
+    # with forgiving spacing and single or double quotes
+    with open("pyproject.toml") as fp:
+        pyproject_txt = fp.read()
+    match = re.search(r"""\n\s*requires-python\s*=\s*['"](.*)['"]""", pyproject_txt)
```

Comment:
3.12 has a toml lib in stdlib if you'd like to use it rather than an evil regex (but happy to stay with regex): https://docs.python.org/3/library/tomllib.html

Reply:
oh yeah I forgot that was included, that's much better!